erasure_code/gf_vect_mad_perf.c

```diff
@@ -34,11 +34,17 @@
 #include "types.h"
 #include "test.h"
 
-//By default, test sse version
 #ifndef FUNCTION_UNDER_TEST
+//By default, test sse version
+#if __x86_64__  || __i386_
```

Comment:
This also should include on windows.
`#if __x86_64__  || __i386__ || _M_X64 || _M_IX86`